### PR TITLE
DAT-4236 | Fix site head markup

### PR DESCRIPTION
**Pull request by rogatty**
## Links

* https://wikia-inc.atlassian.net/browse/DAT-4236

## Description

Unify pre-ember site head markup with the final one. Fixes the issue with article header's position.
Also removes the empty ad-slot placeholder because page jumps anyway and it looked bad.

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: server/app/views/_layouts/error.hbs [hunk 1]
```diff
@@ -19,19 +19,55 @@
...
+				min-height: 3.4375rem;
+			}
+
+			.site-head-wrapper {
+				height: 3.4375rem;
 			}
 
-			.site-head div {
+			.site-head {
+				background-color: #FFF;
```
Comment (Warkot) [inline server/app/views/_layouts/error.hbs]: Are these styles copied from somewhere? If yes, they should just share a selector.

Comment (vforge) [inline server/app/views/_layouts/error.hbs]: Also: we use prefixer, don't we?

And those look like copy/paste from inspector.

Comment (rogatty) [inline server/app/views/_layouts/error.hbs]: This layout is used only on error pages and doesn't include `app.css` so we inline only required styles.

Yes, this is copy/pasted from inspector and no, we don't use prefixer for styles inlined in hbs files.

Comment (Warkot) [inline server/app/views/_layouts/error.hbs]: Makes absolute sense! 👍

**Comment by vforge** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM

**Pull request merged**